iac/stacks/common.py

```
@@ -54,6 +57,227 @@ def __init__(
+                resource_type=ec2.FlowLogResourceType.from_vpc(self.vpc),
+                traffic_type=ec2.FlowLogTrafficType.ALL,
+                destination=ec2.FlowLogDestination.to_s3(
+                    bucket=vpc_flow_logs_bucket,
+                    key_prefix="vpc-flow-logs/",
+                    file_format=ec2.FlowLogFileFormat.PARQUET,
+                    hive_compatible_partitions=True,
+                ),
+            )
+
+            # Create Glue Database
+            glue_database = glue.CfnDatabase(
```

Flake8 (7.3.0)

[error] local variable 'glue_database' is assigned to but never used

(F841)